etc/di.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <virtualType name="IgorTregub\LogQueueToFile\Model\VirtualLoggerHandler" type="Magento\Framework\Logger\Handler\Base">
```

Comment:
It's better to avoid using virtual types. Just for this example - it's ok, but in general using virtual types prevents you to add plugins for your class